projects/element-ng/icon-status/index.ts

```diff
@@ -2,5 +2,10 @@
  * Copyright (c) Siemens 2016 - 2025
  * SPDX-License-Identifier: MIT
  */
+
```

Comment:
I think you this deprecation message is not needed in the index.ts file. You already have one for all the individual symbols.